kubeface/worker_configuration.py

```diff
@@ -17,7 +17,7 @@ def add_args(parser):
             default=DEFAULT.pip)
         parser.add_argument(
             "--kubeface-worker-pip-packages",
-            default=DEFAULT.pip_packages)
+            default=DEFAULT.pip_packages, nargs="+")
```

Comment:
style nit: each arg should be on its own line when the argument list is multiple lines like this